packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3964,9 +4012,21 @@ export default class Meeting extends StatelessWebexPlugin {
               LoggerProxy.logger.info('Meeting:index#join --> enabled to recieve transcription!');
             }
           }
+          // @ts-ignore - config coming from registerPlugin
+          if (this.config.receiveReactions || options.receiveReactions) {
+            if (this.isReactionsSupported()) {
```

Comment:
1) these 2 `if` can be merged into 1
2) you should check the `this.reactions` value as well

```suggestion
          if (this.config.receiveReactions || options.receiveReactions && this.isReactionsSupported() && !this.reactions) {
```

if you have to create a new Reactions instance each time when `joinMeetingOptions` called, then you should unsubscribe the active reactions listener before you start a new one

Reply:
sounds good; Ive separated the platform support conditions from the !this.reactions condition, so that, if the platforms doesn't support it, the corresponding error message is logged.